ajax/libs/response.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "response.js",
+  "version": "0.9.1",
+  "filename": "response.min.js",
+  "description": "Response is an experimental jQuery/Ender/Zepto plugin that gives web designers tools for building responsive websites.",
```

Comment:
Do you think it's a good idea to add the rest of the introduction from GitHub: `It can dynamically swap content based on breakpoints and data attributes.` ?